### PR TITLE
create a specific Grist CSV Viewer section in README

### PR DESCRIPTION
**Pull request by paulfitz**
Clean up language around the CSV Viewer a bit. It is still inherently confusing, there are a lot of options and angles.

I changed CSV popups to default to single page view because it feels like a better default for CSV, and it makes the documentation simpler, one can reasonably leave most options for a separate section without leaving the user with something ugly.

### CODE REVIEW

**Comment by paulfitz** [comment]
Preview at https://github.com/gristlabs/grist-static/blob/paulfitz/csv-viewer-tweaks/README.md#grist-csv-viewer

File: README.md [hunk 1]
```diff
@@ -54,19 +54,45 @@ Browsers don't have native support for Grist [yet :-)] but you can make a little
...
+<head>
+  <script src="https://grist-static.com/csv-viewer.js"></script>
+</head>
+```
+
+Then you can make a button to open CSV from a URL:
+```html
+<button data-grist-csv-open="transactions.csv">View CSV</button>
+```
+The CSV could be a file, or something your site generates for a user dynamically.
```
Comment (dsagal) [inline README.md]: If "something" has to be a URL, I think better to be explicit, i.e. "or a URL of CSV data that your site generates for a user dynamically".

File: README.md [hunk 2]
```diff
@@ -91,39 +115,23 @@ to show either a CSV file are a Grist document of your choice. Full list of opti
...
+  * Use `data-loader` to show a loading spinner while the document is loading. This is enabled by default, pass `data-loader="false"` to disable it.
 
-You can also used a predefined button's classes, like `grist` or `grist-large`, to reuse Grist's 
-default styling.
-
+There are also some predefined button classes, specifically `grist`
+and `grist-large`, that offer Grist's default styling.
 
 For finer control, there is a global `previewInGrist` function with the same API as `bootstrapGrist`,
 but instead of rendering inline it opens a preview in a popup. This might be useful for any dynamically created content or a files that are not available at the time of page load.
```
Comment (dsagal) [inline README.md]: "or a files" -> "or files"

**Review by dsagal — APPROVED** [review]
Looks good except for a typo and one clarification suggestion. Thank you!

**Pull request merged**